src/main/java/somethingrandom/usecase/delete/DeleteItemInteractor.java

```diff
@@ -0,0 +1,28 @@
+package somethingrandom.usecase.delete;
+
+import somethingrandom.usecase.DataAccessException;
+
+import java.util.UUID;
+
+public class DeleteItemInteractor implements DeleteItemInputBoundary{
+    private final DeleteItemDataAccessInterface dao;
+    private final DeleteItemOutputBoundary presenter;
+
+    public DeleteItemInteractor(DeleteItemDataAccessInterface dao, DeleteItemOutputBoundary presenter) {
+        this.dao = dao;
+        this.presenter = presenter;
+    }
+
+    public void execute(UUID id) {
+        try {
+            String message = dao.delete(id);
+            if (message == null){
+                presenter.presentFailureView("Item not found");
+            } else {
+                presenter.presentSuccessView(message);
+            }
```

Comment:
This is subjective, but is there any need to distinguish between 'not found' (which should never really happen anyways, but good to check) and 'deleted'? In either case, the requested item isn't there anymore.